modules/ui/map3d_viewer.js

```diff
@@ -277,7 +274,6 @@ export function uiMap3dViewer(context) {
     map.on('draw', () => redraw());
     map.on('move', () => redraw());
     context.keybinding().on([uiCmd('⌘' + l10n.t('background.3dmap.key'))], toggle);
-    urlhash.on('hashchange', toggle);
```

Comment:
My initial thought is that it would be a shame to lose the connection between the `map3d` urlhash param, and the state of the viewer, but looking elsewhere around the code it seems like this was never really set up correctly so we aren't losing anything.

I expect it to work like this:
- checking/unchecking the box updates the `map3d=true` to the urlhash  
(something like `urlhash.setParam('map3d', !_isHidden ? 'true' : null)`)
- changing the urlhash updates the state of the box and viewer
(something like `urlhash.on('hashchange', doStuff)`)

Basically we should just look at how `PhotoSystem` or `ImagerySystem` works and make `Map3dSystem` work more like those.

Anyway, I'm going to just merge this because we need to patch the bug, but we should improve this later.